src/components/interface/AudioPlay.jsx

```diff
@@ -33,20 +89,23 @@ class AudioPlayer extends React.Component {
         <div style={{ alignSelf: 'center', flex: 3 }}>
           {fileName}
         </div>
-        <div style={{ alignSelf: 'center', flex: 2 }}>
+        <div style={{ alignSelf: 'center', flex: 0 }}>
           {time}
         </div>
-        <div>
-          <PlayCircleOutlineIcon style={{ float: 'right', marginLeft: '25px' }} color="primary" />
+        <div style={{ flex: 0 }}>
+          {
```

Comment:
And a one last comment, why are these JSX objects wrapped in `{ }`?

Typically we use `{ }` evaluating JS inside of these React nodes.

Reply:
I think wrapping the JSX objects in `{ }` was for organisation. I have removed them and alphabetised the properties.

Reply:
`{ }`  were left over from an old comment. Thanks!